websites/J/JSR/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.13",
+  "apiVersion": 1,
+  "author": {
+    "id": "357461892526243840",
+    "name": "litomore"
+  },
+  "service": "JSR",
+  "description": {
+    "en": "TODO"
```

Comment:
The description is marked as "TODO". Probably fixing this is a good idea!

Reply:
Wow, good catch! I just missed this one! Thanks for pointing out!